<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputSlatepackDlg</class>
 <widget class="QDialog" name="InputSlatepackDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="slatepackEdit">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>90</y>
     <width>741</width>
     <height>256</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>6000</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please paste the slatepack content below</string>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="title">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>20</y>
     <width>701</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Slatepack</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="continueButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="slatepack_status">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>355</y>
     <width>741</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypted, from XXXXXXXX</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
